---
title: ""
output: 
  word_document:
  latex_engine: xelatex
fontsize: 12pt
geometry: margin = 1in
bibliography: sal_manuscript.bib
csl: msystems.csl
header-includes:
- \usepackage{lineno}
- \usepackage{setspace}
- \doublespacing

---

**Title**: Prebiotic galactooligosaccharide feed modifies the chicken gut microbiota 
to efficiently clear *Salmonella*

**Running Head:** Prebiotic GOS accelerates *Salmonella* clearance

**Authors:** Philip J. Richards^1^, Abeer Almutrafy^1^, Lu Liang^1^, 
Geraldine M. Flaujac Lafontaine^1^, Elizabeth King^1^, Neville M. Fish^2^, 
Amber J. Connerton^1^, Phillippa L. Connerton^1^ and Ian F. Connerton^1,3^ 

^1^Division of Microbiology, Brewing and Biotechnology, School of Biosciences, 
University of Nottingham, Loughborough, Leicestershire LE12 5RD, UK  

^2^Saputo Dairy UK (c/o Simon Hunt), Saputo Dairy UK Innovation Centre, 
Harper Adams University, Edgmond, Newport, TF10 8NB, UK

^3^Lead Contact

*Correspondence: ian.connerton@nottingham.ac.uk

Key words: prebiotic, GOS, galactooligosaccharide, chicken, Salmonella, SCFA,
propionate, valerate, microbiome, microbiota

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

set.seed(1216)

# packages
library("readr") 
library("tidyr")
library("stringr")

# input
SE_enrich_raw_path <- "resources/zootechnical/late_SE_enrich.csv"
error_path <- "results/mothur/mock.error.count"

```

Abstract word count: 260 words  
Importance word count: 75 words  
Text word count: 4996 words  

```{r SCFA data, include=FALSE}
# draw Figure 4
source("code/F4_scfa.R", echo=TRUE)

# get p-values for inline reporting
prop_28_jGOS_Salm <- scfa_pw %>% 
                       filter(SCFA=="Propionate", age=="28", group1=="jGOS x SE") %>% 
                       pull(p.value) %>% 
                       max()
 
prop_35_jGOS_Salm <- scfa_pw %>% 
                       filter(SCFA=="Propionate", age=="35", group1=="jGOS x SE") %>% 
                       pull(p.value) %>% 
                       max()
 
prop_28_MOCKvsSE <- scfa_pw %>% 
                       filter(SCFA=="Propionate", age=="28", 
                              group2=="ctl x Mock", group1 =="ctl x SE") %>% 
                       pull(p.value)
 
prop_35_MOCKvsSE <- scfa_pw %>% 
                       filter(SCFA=="Propionate", age=="35", 
                              group2=="ctl x Mock", group1=="ctl x SE") %>% 
                       pull(p.value)
 
prop_28_ctlsalmVSjgosmock <- scfa_pw %>% 
                                 filter(SCFA=="Propionate", age=="28", 
                                        group1=="jGOS x Mock", group2=="ctl x SE") %>% 
                                 pull(p.value)

prop_35_ctlsalmVSjgosmock <- scfa_pw %>% 
                                 filter(SCFA=="Propionate", age=="35", 
                                        group1=="jGOS x Mock", group2=="ctl x SE") %>% 
                                 pull(p.value)

valerate_35_MOCKvsSE <- scfa_pw %>% 
                               filter(SCFA=="Valerate", age=="35", 
                                      group1=="ctl x SE", group2 == "ctl x Mock") %>% 
                               pull(p.value)

 valerate_28_jGOS_Salm <- scfa_pw %>% 
                             filter(SCFA=="Valerate", age=="28", group1=="jGOS x SE") %>% 
                             pull(p.value) %>% 
                             max()
 
 valerate_35_jGOS_Salm <- scfa_pw %>% 
                             filter(SCFA=="Valerate", age=="35", group1=="jGOS x SE") %>% 
                             pull(p.value) %>% 
                             max()
 
 valerate_35_ctlsalmVSjgosmock <- scfa_pw %>% 
                                 filter(SCFA=="Valerate", age=="35", 
                                        group1=="jGOS x Mock", group2=="ctl x SE") %>% 
                                 pull(p.value)
 

```

## Abstract
Chicken meat is contaminated with *Salmonella* from the gut of infected 
chickens during slaughter. Eradication of *Salmonella* from broiler chickens 
through hygiene measures and/or vaccination is not cost-effective; complementary 
approaches are required. A mature gut microbiota obstructs *Salmonella* infection 
in chickens and deliberate fortification of colonization resistance through 
prebiotic feed formulations would benefit public health and poultry production. 
Prebiotic galactooligosaccharides hastens *Salmonella* clearance from the gut of 
infected chickens. To better understand the role of galactooligosaccharides in 
colonization resistance broiler chickens were raised on a wheat-soybean meal-based 
feed, with or without galactooligosaccharides for the first 24-days of life. 
Chickens were orally challenged with *Salmonella enterica* serovar Enteritidis 
at 20-days and the effect of supplementary galactooligosaccharides characterized 
by profiling *Salmonella* colonization, gut microbiota, innate immune response 
and cecal SCFA concentrations. Exposure to dietary galactooligosaccharides shortened 
the time to clear *S*. Enteritidis from the ceca. Differential abundance analysis 
of the cecal microbiota associated *Salmonella*-challenge with a bacterial taxon 
belonging to the Acidaminococcaceae family (*p* < 0.005). Increased cecal 
concentrations of the short-chain fatty acids propionate and valerate were measured 
in *Salmonella*-challenged chickens sustained on either control or 
galactooligosaccharide-supplemented feed relative to mock-challenged controls; 
but far greater concentrations were detected in chickens fed a 
galactooligosaccharide-supplemented diet in early life. The abundance of the 
Acidaminococcaceae taxon exhibited a positive correlation with the cecal concentrations 
of propionate ($\rho$ = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age=="35 (15)", SCFA == "Propionate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age == "35 (15)", SCFA == "Propionate") %>% pull(q) %>% round(.,3)`) and valerate ($\rho$ = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age=="35 (15)", SCFA == "Valerate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age=="35 (15)", SCFA == "Valerate") %>% pull(q) %>% round(.,3)`). 
The absence of cecal proinflammatory transcriptional responses suggest that the rapid 
*Salmonella*-clearance observed for the galactooligosaccharide-supplemented 
diet was not linked to innate immune function.

## Importance
Work presented here identifies bacterial taxa responsible for colonization resistance 
to *Salmonella* in broiler chickens. Deliberate cultivation of these taxa with 
prebiotic galactooligosaccharide has potential as a straight-forward, safe and 
cost-effective intervention against *Salmonella*. We hypothesise that catabolism 
of galactooligosaccharide and its breakdown products by indigenous microorganisms 
colonizing the chicken gut produce excess levels of propionate and valerate. In 
the absence of gross inflammation propionate and valerate are inimical to *Salmonella* 
and hasten intestinal clearance.

## Introduction
*Salmonella enterica* is a leading cause of diarrheal disease in humans [@troeger2017estimates] 
and *Salmonella enterica* subsp. *enterica* serovar Enteritidis in particular is 
a key causative agent of salmonellosis commonly associated with contaminated chicken 
meat and eggs [@hendriksen2011global; @ferrari2019worldwide]. Broiler chickens are 
birds specifically bred for meat production, and although vaccination of maternal broilers 
and stringent hygiene practices can exclude vertical transmission of *Salmonella*, 
factors such as short lifespan (typically 35-42 days) and cost make the vaccination 
of production broiler flocks unattractive to primary producers [@european2004opinion]. Therefore, 
most broiler chickens raised in intensive farming systems are susceptible to *Salmonella* 
colonization through lapses in hygiene control, which includes introduction via 
contaminated feed or the entry of *Salmonella* from the outside environment [@le2010prevalence]. 
Husbandry practices that reduce the likelihood of flock colonization in the 
event of accidental *Salmonella* exposure are of interest to the poultry industry.

Colonization resistance is the exclusion of orally introduced organisms by the 
resident microbiota [@van1971colres] and it has long been established that this
phenomenon has a key role in protecting chickens from *Salmonella* colonization.
In the absence of maternally inherited anti-*Salmonella* IgY [@dasilva2022maternal], 
oral challenge of newly hatched chicks with *Salmonella* will effectively colonize 
the gastrointestinal tract with high densities of *Salmonella*. However, from ~7 
days of age a challenge of similar magnitude leads to less frequent successful 
colonization with lower levels of *Salmonella*; often without the overt presentation 
of disease symptoms [@wigley2014salmonella; @mon2021tolerogenic; @desmidt1997pathogenesis; @beal2004age; @varmuzova2016composition]. 
Colonization resistance can be deliberately produced in young chicks (<7-days-old) 
through exposure to the gut microbiota of mature birds [@rogers2020microbiome; @nurmi1973new; @varmuzova2016composition]. 
However, hygiene interventions put in place in intensive production systems to prevent 
vertical pathogen transmission blocks exposure of chicks to the commensal and/or 
synergistic maternal gut microbiota thereby potentially increasing the likelihood of 
gastrointestinal colonization by *Salmonella* [@rychlik2020composition]. 

Prebiotics are defined as substrates that are selectively utilized by microorganisms 
conferring a health benefit to the host [@gibson2017prebiotic]. Deliberate modification 
of the chicken gut microbiota using prebiotics to bring about the gastrointestinal exclusion 
of *Salmonella* is desirable due to the relatively light regulatory burden they 
face and the ease of application relative to other feed supplements. Commercial 
prebiotic galactooligosaccharides (GOS) are enzymatically synthesized from lactose 
by $\beta$-galactosidase catalyzed trans-galactosylation, which produces $\beta$(1→3), 
$\beta$(1→4) and $\beta$(1→6) linked galactose units with a terminal glucose as 
a mixture of oligosaccharides composed of 2 to 8 monomeric units [@vanleeuwen2016gosstructure]. 
GOS have been shown to accelerate the clearance of *S.* Typhimurium from the ceca of layer 
hens [@azcarate2018attenuated]. Parallel observations that the supplementation of 
drinking water with a GOS preparation led to increased cecal propionate levels in 
rats [@fernandez2018galacto] and that competitive exclusion of *S.* 
Typhimurium in mice is mediated by propionate generated by *Bacteroides* [@jacobson2018propionate] 
suggest a common mechanistic role for propionate. 

Consumer demand for chicken meat remains high but competition between broiler chicken producers, in a market 
with rising costs of raw materials and energy, necessitates that production 
costs are tightly controlled. With the financial burden associated with dietary 
GOS inclusion in mind, supplementary GOS was included in the lower volume starter 
(0 - 10 days old) and grower (11 - 24 days old) diets at 2.32% (w/w) and 1.14% 
(w/w) respectively. We report that dietary GOS and *Salmonella*-challenge 
shape the mature broiler chicken cecal microbiota to affect the 
abundances of short-chain fatty acids, which act
to efficiently clear *Salmonella* colonization.

## Results

### Dietary GOS in early life hastens *Salmonella* cecal clearance from broiler chickens
```{r salmonella colonization, include=FALSE}
# draw Figure 2
source("code/F2_SE_effect.R", echo=TRUE)

# read in data
SE_enrich_raw <- read_csv(SE_enrich_raw_path)

# format
SE_enrich <- SE_enrich_raw %>% 
                      separate(col = "bird", sep = "_", 
                               into = c("group","age","bird")) %>% 
                      group_by(group,age,site) %>% 
                      summarise(n = n(),
                                pos = sum(str_count(status, 'pos')))

```
Azcarate-Peril *et al.* [@azcarate2018attenuated] reported that supplementation of a 
corn-soybean based-feed with 0.55% w/w GOS hastened clearance of *S.* Typhimurium 
from the ceca of white leghorn layer chickens. However, the effectiveness of 
prebiotics in poultry is dependent on the prebiotic formulation and the microbes
colonizing the gastrointestinal tract [@ricke2020prebiotics]. To test if GOS 
promoted the clearance of *Salmonella* from broiler chickens sustained on an 
alternate standard feed composition we orally challenged 20-day-old 
Ross 308 broiler chickens raised on wheat-soybean meal-based feed with *Salmonella*
Enteritidis. Chickens were fed either a control diet (ctl group) or a diet with
supplementary GOS inclusion for juvenile birds up until 24 days old (jGOS group). 
As age is 
known to influence both the persistence of *Salmonella* infection and the ecology 
of the gut microbiota [@beal2004age; @richards2019development; @ocejo2019productive];
intestinal contents and tissue samples were collected from birds sacrificed over 
a period of 15 days post-infection (dpi) with the endpoint at 35 days being a 
commonplace slaughter age. Therefore, *Salmonella* colonization levels are 
reported with respect to both the age of the birds and the time duration post 
challenge. No *Salmonella* were recovered from the intestinal contents or organs 
of the mock-challenged chickens fed either the control or the GOS-supplemented 
diet.

At 22 and 24 days-old, equal to 2 and 4 dpi, similar levels of viable 
*Salmonella* were cultured from the cecal contents of *Salmonella* challenged 
chickens raised on either control or GOS-supplemented feed (*p* $\geq$ `r round(SE_counts_stats %>% filter(Age == 22 | Age == 24) %>% pull(p.value) %>% min(), 3)`; Figure 2A).
However, by 28 days-old (8 dpi) the levels of *Salmonella* isolated from the cecal 
contents of chickens on the control diet were greater than those recovered 
from jGOS birds (*p* = `r  round(SE_counts_stats %>% filter(Age == 28) %>% pull(p.value), 3)`). 
Correspondingly, *Salmonella* could only be directly enumerated from `r SE_counts %>% filter(.[[2]]==28,.[[4]]=="GOS", .[[5]]>1.7) %>% nrow()` of the `r SE_counts %>% filter(.[[2]] == 28,.[[4]]=="GOS") %>% nrow()` chickens 
from the jGOS group, with recovery of *Salmonella* from the cecal content of the 
remaining `r SE_counts %>% filter(.[[2]] == 28,.[[4]]=="GOS", .[[5]]<=1.7) %>% nrow()` 
birds dependent on enrichment culture methods. *Salmonella* was directly 
enumerated from `r SE_counts %>% filter(.[[2]] == 28,.[[4]]=="ctl", .[[5]]>1.7) %>% nrow()` 
of the `r SE_counts %>% filter(.[[2]] == 28,.[[4]]=="ctl") %>% nrow()` chickens on 
the control diet, with enrichment only required to recover *Salmonella* from 
`r SE_counts %>% filter(.[[2]] == 28,.[[4]]=="ctl", .[[5]]<=1.7) %>% nrow()` birds; 
indicating that higher levels of *Salmonella* were present in chickens raised on the control
feed (Figure 2A). One week later, at 35-days old (15 dpi), the levels of *Salmonella* 
isolated from chickens from the jGOS group remained lower than those in chickens on the control diet 
(*p* = `r round(SE_counts_stats %>% filter(Age == 35) %>% pull(p.value), 3)`). 
At this time *Salmonella* was directly enumerated from `r SE_counts %>% filter(.[[2]] == 35,.[[4]]=="ctl", .[[5]]>1.7) %>% nrow()` of the `r SE_counts %>% filter(.[[2]] == 35, .[[4]]=="ctl") %>% nrow()` chickens 
on the control diet but was not recoverable from the cecal contents of any of the 
`r SE_counts %>% filter(.[[2]] == 35,.[[4]]=="GOS") %>% nrow()` chickens from the jGOS 
group even with the aid of enrichment (Figure 2A). *S*. Enteritidis is frequently
found in the spleen and liver of chickens after oral challenge [@zeng2018distribution].
By 4 dpi *Salmonella* was detected by enrichment from the spleens and livers of 
`r SE_enrich %>% filter(age==22) %>% pull(n) %>% unique()` of 
`r SE_enrich %>% filter(age==22) %>% pull(n) %>% min()` birds from the challenged 
chickens of both the control and jGOS diet groups, but by 15 dpi (35 days old)
*Salmonella* could only be enriched from the livers of `r SE_enrich %>% filter(age==35,site=="Liver") %>% pull(pos) %>% unique()` 
of `r SE_enrich %>% filter(age==35,site=="Liver") %>% pull(n) %>% unique()` chickens
in each group (Table S1). Correspondingly, *Salmonella* was recovered from the 
spleens of `r SE_enrich %>% filter(age==35,site=="Spleen", group=="Gp 3") %>% pull(pos)` of `r SE_enrich %>% filter(age==35,site=="Spleen", group=="Gp 3") %>% pull(n)` 
chickens from the control group and `r SE_enrich %>% filter(age==35,site=="Spleen", group=="Gp 4") %>% pull(pos)` of 
`r SE_enrich %>% filter(age==35,site=="Spleen", group=="Gp 4") %>% pull(n)` chickens from the jGOS 
group. 

Previously we have reported that supplementing feed with GOS improves 
the growth rate of *Salmonella*-free chickens on a wheat-soybean meal-based 
feed [@richards2020gos]. In the results presented here the body weights of 
*Salmonella* challenged chickens in the jGOS group were greater than 
mock-challenged chickens on the calorie-balanced control diet at `r weight_pw %>% filter(p.value <= 0.05) %>% pull(Age)` days 
old (*p* = `r weight_pw %>% filter(p.value <= 0.05) %>% pull(p.value) |> round(3)`; Figure 2B), 
corresponding with the end of the period during which supplementary GOS was 
added to the diet. No significant difference in the body weights was detected at 
28 or 35 days, which included the SE-challenged birds that did not suffer any growth check. 
These findings indicate that whilst dietary inclusion of GOS in early life hastened the clearance of 
*Salmonella*, the inclusion did not effect the body weight at 35 days old.

### Dietary GOS in early life affects cecal microbiota diversity following *Salmonella* Enteritidis challenge 
```{r rarefaction, include=FALSE}
# draw Supplementary Figure 1
source("code/SF1_rarefaction.R", echo=TRUE)

```
```{r microbiota, include=FALSE}
# draw Figure 3
source("code/F3_host_microbe.R", echo=TRUE)
# draw Supplementary Figure 2
source("code/SF2_shannon.R", echo=TRUE)

# format stats output for inline reporting
dunnflat <- dunn %>% 
                    mutate(model = map(dunntest, ~(.)$res)) %>% 
                    unnest(cols=model) %>% 
                    select(-c(data,dunntest))

shannon_dunnflat <- shannon_dunn %>% 
                      mutate(model = map(dunntest, ~(.)$res)) %>% 
                      unnest(cols=model) %>% 
                      select(-c(data,dunntest))

```
Mon *et al.* @mon2020integrative reported that *S*. Enteritidis colonization of the 
cecum of two-week old layer chickens was accompanied by changes in the bacterial 
microbiota without any indication of disease. We were interested to know if the 
cecal microbiota of broiler chickens was similarly affected by *Salmonella* 
challenge and whether supplementary GOS in early life might disrupt these changes 
and potentially affect *Salmonella* clearance. The cecal microbiota of the 
experimental chickens was profiled using 16S rRNA amplicon sequencing to an adequate 
sequencing depth (Figure S1). In agreement with Mon *et al.* @mon2020integrative, 
we observed that whereas the $\alpha$-diversity of the cecal community of chickens 
sustained on the control feed was unchanged by SE-challenge (Inverse Simpson Index: *p* $\geq$ `r round(dunnflat %>% filter(Comparison=="ctl sal - ctl unc") %>% pull(P.adj),3)`, Figure 3A; Shannon Entropy: *p* $\geq$ `r round(shannon_dunnflat %>% filter(Comparison=="ctl sal - ctl unc") %>% pull(P.adj),3)`, Figure S2), changes the $\beta$-diversity of these bacterial
communities did accompany *Salmonella* colonization, as revealed by analysis of 
the Bray-Curtis dissimilarity (Figure 3B). However, when supplementary GOS was 
introduced to the diet of juvenile chickens, by 15 dpi the alpha-diversity of the 
bacterial communities in the caeca was significantly lower in SE-challenged birds 
than that of all other treatment groups (Inverse Simpson Index: *p* $\leq$ `r round(dunnflat %>% filter(str_detect(Comparison, "gos sal")==TRUE) %>% pull(P.adj) %>% max(),3)`, Figure 3A; Shannon Entropy: *p* $\leq$ `r round(shannon_dunnflat %>% filter(str_detect(Comparison, "gos sal")==TRUE) %>% pull(P.adj) %>% max(),3)`, Figure S2). 
Correspondingly, chickens on the jGOS feeding regime also exhibited a more profound 
change in the $\beta$-diversity of the cecal microbiota following SE-challenge
than mock-challenged chickens from the jGOS group, and both the SE- and mock-challenged 
chickens in the control-feed group (Figure 3A). Changes in $\beta$-diversity are 
particularly evident from 8 dpi onwards. These data show that sustaining broiler 
chickens on a GOS supplemented diet up until they are 24 days old affects changes 
in the cecal microbiota caused by *Salmonella* challenge 10 days after supplementary
GOS was withdrawn.

By comparison with other forms of analysis, Nearing *et al.* [@nearing2022daamethods] showed 
that ALDEx2 [@fernandes2013anova; @fernandes2014unifying; @gloor2016displaying] 
and ANCOM-based approaches [@lin2020analysis; @lin2022linear] delivered the most 
conservative results for the identification of discriminative taxa, with a lower 
risk of false positives. Confidence that the OTUs identified by the procedure 
are not the result of bacterial DNA contamination of the kit or reagents was 
evident from their absence or low abundance in the ‘kitome’ negative control 
(Table S2). ALDEx2 and ANCOM-based packages were used here to determine 
those OTUs associated with either SE-challenge or inclusion of supplementary GOS,
the greatest *p*-value returned by either package is reported here.  SE-challenge resulted in the increased abundance of `r df %>% filter(chal_fill=="Positive LFC") %>% nrow()` bacterial OTUs, and
in particular `r df %>% filter(OTU=="Otu0007") %>% pull(id) %>% str_extract(string = ., pattern = "^\\S*")` (log~10~ fold-change = `r df %>% filter(id == "Acidaminococcaceae_unclassified (Otu0007)") %>% pull(lfc_challengeSE) %>% round(.,2)`, *p*-value < 0.005).
The remaining OTUs that were positively associated with SE-challenge, but with smaller
effect size were as follows: 
*Megamonas* spp. (Otu0006; log~10~ fold-change = `r df %>% filter(id == "Megamonas (Otu0006)") %>% pull(lfc_challengeSE) %>% round(.,2)`, *p*-value < 0.005),
*Bifidobacterium* spp. (Otu0015; log~10~ fold-change = `r df %>% filter(id == "Bifidobacterium (Otu0015)") %>% pull(lfc_challengeSE) %>% round(.,2)`, *p*-value < 0.005),
*Bacteroides* spp. (Otu0027; log~10~ fold-change = `r df %>% filter(id == "Bacteroides (Otu0027)") %>% pull(lfc_challengeSE) %>% round(.,2)`, *p*-value < 0.005), *Lactobacillus* spp. (Otu0061; log~10~ fold-change = `r df %>% filter(id == "Lactobacillus (Otu0061)") %>% pull(lfc_challengeSE) %>% round(.,2)`, *p*-value $\leq$ `r df %>% filter(id == "Lactobacillus (Otu0061)") %>% select('challengeSE:pval.padj', q_challengeSE) %>% pivot_longer(names_to = "daa", cols = c('challengeSE:pval.padj', q_challengeSE)) %>% pull(value) %>% max() %>% round(.,3)`) and
*Olsenella* spp. (Otu0082; log~10~ fold-change = `r df %>% filter(id == "Olsenella (Otu0082)") %>% pull(lfc_challengeSE) %>% round(.,2)`, *p*-value $\leq$ `r df %>% filter(id == "Olsenella (Otu0082)") %>% select('challengeSE:pval.padj', q_challengeSE) %>% pivot_longer(names_to = "daa", cols = c('challengeSE:pval.padj', q_challengeSE)) %>% pull(value) %>% max() %>% round(.,3)`) 
(Figures 3C). No OTUs showed significant reductions in abundance in response to SE-challenge.

Across all time points the abundance of `r df %>% filter(diet_fill=="Positive LFC") %>% nrow()`
taxa exhibited a positive log FC in response to the jGOS feeding regime: *Megamonas* spp. 
(Otu0006; log~10~ fold-change = `r df %>% filter(id == "Megamonas (Otu0006)") %>% pull(lfc_dietgos) %>% round(.,2)`; *p*-value $\leq$ `r df %>% filter(id == "Megamonas (Otu0006)") %>% select(q_dietgos, 'dietgos:pval.padj') %>% pivot_longer(names_to = "daa", cols = c(q_dietgos,'dietgos:pval.padj')) %>% pull(value) %>% max() %>% round(.,3)`), *Lactobacillus* spp. (Otu0061; log~10~ fold-change = `r df %>% filter(id == "Lactobacillus (Otu0061)") %>% pull(lfc_dietgos) %>% round(.,2)`; *p*-value $\leq$ `r df %>% filter(id == "Lactobacillus (Otu0061)") %>% select(q_dietgos, 'dietgos:pval.padj') %>% pivot_longer(names_to = "daa", cols = c(q_dietgos,'dietgos:pval.padj')) %>% pull(value) %>% max() %>% round(., 3)`) and
*Enterococcus* spp. (Otu0063; log~10~ fold-change = `r df %>% filter(id == "Enterococcus (Otu0063)") %>% pull(lfc_dietgos) %>% round(.,2)`; *p*-value $\leq$ `r df %>% filter(id == "Enterococcus (Otu0063)") %>% select(q_dietgos, 'dietgos:pval.padj') %>% pivot_longer(names_to = "daa", cols = c(q_dietgos,'dietgos:pval.padj')) %>% pull(value) %>% max() %>% round(., 3)`) (Figures 3D). Both  *Megamonas* spp. (Otu0006) and *Lactobacillus* spp. (Otu0061)
were increased in chickens on the jGOS diet whilst also showing association with 
SE-challenge.  Ruminococcaceae_unclassified spp. (Otu0018; log~10~ fold-change = `r df %>% filter(id == "Ruminococcaceae_unclassified (Otu0018)") %>% pull(lfc_dietgos) %>% round(.,2)`; 
*p*-value $\leq$ `r df %>% filter(id == "Ruminococcaceae_unclassified (Otu0018)") %>% select(q_dietgos, 'dietgos:pval.padj') %>% pivot_longer(names_to = "daa", cols = c(q_dietgos,'dietgos:pval.padj')) %>% pull(value) %>% max() %>% round(., 3)`) was the sole taxa reduced by the jGOS diet (Figure 3D). 
In contrast to the blooms in the relative abundance of bacteria of the phylum Proteobacteria
reported to accompany *Salmonella*-mediated inflammation in mouse infection models
[@stecher2007salmonella], no taxa classified as Proteobacteria were observed to 
increase in abundance in response to *Salmonella*-challenge in the chicken intestinal
microbiota (Figures 3C), possibly due to the absence of gross inflammation.

```{r host:microbiota interaction, include=FALSE}
# draw Supplementary Figure S2
source("code/SF3_host.R", echo=TRUE)

```
*Salmonella* colonization and clearance in chickens is linked to innate and
adaptive immunological responses. Innate immune responses evident in, but not 
exclusively, the ileum and ceca are reported to involve IL-17, IL-22, IL-8, 
IFN-$\gamma$ and IL-1$\beta$ [@withanage2005cytokine; @crhanova2011immune; 
@setta2012early]. We therefore profiled the expression of a panel of these and other immune 
function-linked genes in cecal tissues to determine if the influence of GOS on the 
microbiota was associated with changes in host immunity. The supplementation of juvenile feed with GOS differentiated the 
host response of chickens to SE colonization in the cecum at 22 days old (2 dpi), 
with up-regulation of IL-8L2 (*p* = `r FC.stats %>% filter(Age=="22", str_detect(gene, pattern="8L2")) %>% pull(p_adjust) %>% round(3)`) 
observed in SE-challenged birds on the control diet (Figure 3E), alongside up-regulation 
of STAT3 (*p* = `r FC.stats %>% filter(Age==22, gene=="STAT3") %>% pull(p_adjust) %>% round(3)`) and TGIF1 (*p* = `r FC.stats %>% filter(Age==22, gene=="TGIF1") %>% pull(p_adjust) %>% round(3)`) (Figure S3). 

```{r serum, include=FALSE}
source("code/SF4_serum.R", echo=TRUE)

```

Increases in anti-*Salmonella* serum antibodies were detected by ELISA in response 
to *Salmonella* challenge. *Salmonella* infected chickens exhibited an increase 
in IgY compared to the non-challenged birds at 8 dpi (control diet *p* = 0.049; 
jGOS supplemented diet *p* = 0.042) and increases for IgY and IgA at 15 dpi 
(control diet IgY *p* = 0.024 and IgA *p* = 0.008; jGOS supplemented diet IgY 
*p* = 0.004 and IgA *p* = 0.006). No significant differences were observed between 
*Salmonella* infected chickens on the control and jGOS supplemented diets for 
either IgY or IgA at any of the sampling time points (Figure S4).

Diarrhea and mild inflammation of the ileum wall are associated with *Salmonella*
infection of chickens @withanage2005cytokine. In the study reported here chickens 
challenged at 20-days-old did not exhibit any change in behavior or clinical 
signs associated with salmonellosis in chickens or exhibit any gross pathology 
post-mortem. Hematoxylin and eosin-stained histological sections of ileal tissues 
revealed no significant differences in villus height at any time 
point (*p^adj^* $\geq$ `r min(villus_TukeyHSD$adj.p.value) %>% round(3)`, 
Figure 3F).  However, we note that *Salmonella* could be detected 
by enrichment in the spleens and livers of infected birds at 2, 4, 8 and 15 dpi 
(Table S1) but with minimal mononuclear cell infiltration of the hepatic parenchyma 
evident. The incidence of *Salmonella* recoverable from the liver 
was reduced to `r SE_enrich %>% filter(age==35,site=="Liver") %>% pull(pos) %>% unique()`
birds from each group by 15 dpi (35 days-old) with no observable pathology. 

### Cecal SCFA concentrations are correlated with Acidaminococcaceae
Short chain fatty acids (SCFA) are produced in the gut through the microbial breakdown 
of carbohydrates [@flint2012microbial], and the antimicrobial potential of SCFA in 
the chicken gut has been recognized for some time [@ricke2003perspectives]. 
Given that supplementation of drinking water with a GOS preparation increases 
cecal propionate levels in rats [@fernandez2018galacto] and that increased levels 
of gastrointestinal propionate have been established as protecting against 
*Salmonella* infection in mice [@berchieri1996reduction], we investigated whether 
shifts in SCFA were associated with provision of GOS-supplemented feed in 
early-life and/or *Salmonella* challenge of broiler chickens. Concentrations of 
propionate and 8 other SCFAs (acetic acid, isobutyric acid, butyrate, 2-methyl 
butyric acid, isovaleric acid, valerate, hexanoic acid and lactic acid) were 
measured in luminal cecal contents, but significant differences in SCFA levels 
between cohorts were only evident for propionate, butyrate and valerate. 
A similar trend was noted for each of these SCFAs with greater concentrations 
detected in *Salmonella*-challenged chickens, and most profoundly in 
*Salmonella*-challenged chickens in the jGOS group. At 8 dpi (28 days-old) 
butyrate levels in SE-challenged, jGOS group chickens (mean `r round(scfa_long %>% filter(SCFA=="Butyrate", age=="28", cohort=="jGOS x SE") %>% pull(Concentration) %>% mean(),1)` 
mM) were greater than unchallenged chickens raised on control feed (mean `r round(scfa_long %>% filter(SCFA=="Butyrate", age=="28", cohort=="ctl x Mock") %>% pull(Concentration) %>% mean(),1)` 
mM; *p* = `r scfa_pw %>% filter(SCFA=="Butyrate", age =="28") %>% pull(p.value) %>% round(3)`; 
Figure 4A), but at this time all groups were better differentiated by propionate 
and valerate concentrations which were significantly higher in SE-challenged 
chickens in the jGOS group than any other group (propionate 
*p* $\leq$ `r round(prop_28_jGOS_Salm,3)`; valerate *p* $\leq$ `r round(valerate_28_jGOS_Salm,3)`). 
At 35 days-old the concentrations of propionate and valerate were greater in the 
SE-challenged chickens in the jGOS group than mock-challenged chickens on the 
same dietary regimen (propionate *p* $\leq$ `r round(prop_35_jGOS_Salm,3)`; 
valerate *p* $\leq$ `r round(valerate_35_jGOS_Salm,3)`). Correspondingly the levels 
of propionate measured for the SE-challenged 
chickens on the control diet exceeded those for the mock-challenged chickens on 
the control diet at 28 and 35 days (*p* $\leq$ `r round(prop_28_MOCKvsSE,3)` 
and *p* $\leq$ `r round(prop_35_MOCKvsSE,3)` respectively), and the levels of 
valerate were greater in SE-challenged chickens on the control diet than 
mock-challenged chickens on the control diet at 35 days (*p* $\leq$ `r round(valerate_35_MOCKvsSE,3)`). 
Notably propionate was also detected at greater levels in SE-challenged chickens 
sustained on the control diet than the mock-challenged jGOS chickens at 28 and 
35 days, and similarly valerate concentrations were greater in the SE-challenged
chickens on the control diet than the mock-challenged jGOS group at 35 days
(*p* $\leq$ `r max(prop_28_ctlsalmVSjgosmock, prop_35_ctlsalmVSjgosmock, valerate_35_ctlsalmVSjgosmock) %>% round(3)`). These observations suggest that the
increased levels of cecal propionate and valerate are a consequence of the SE-challenge, 
but greater concentrations of these SCFAs can be achieved in chickens belonging 
to the jGOS group. 

```{r OTU vs SCFA, include=FALSE}
# draw Supplementary Figure S5
source("code/SF5_scfa_OTU.R", echo=TRUE)

# get OTUs in agreement
corob <- abund_corr |> 
               select(c(age, SCFA, taxon, estimate, q)) |> 
               left_join(OTU_corr |> select(c(age, SCFA, OTU, estimate, q)), 
                         suffix = c("_bc", "_ra"),
                         by = c("taxon" = "OTU", "SCFA", "age"), 
                         keep = TRUE
                         ) |> 
               mutate(estimate_ra = if_else(estimate_ra < 0, 0, 1),
                      estimate_bc = if_else(estimate_bc < 0, 0, 1)) |> 
               filter(q_ra <= 0.05, 
                      q_bc <= 0.05,
                      estimate_ra == estimate_bc) 

```
A correlation analysis was undertaken on the observations of the levels of propionate, 
butyrate and valerate and the abundance of the OTUs from the ceca of each individual 
chicken. Due to the recognized drawbacks inherent in proportion-based approaches 
[@quinn2021critique], bias-corrected OTU abundances were generated using the 
ANCOM-BC package and the association between these abundances and the 
concentration of each SCFA was calculated by Spearman's rank coefficient ($\rho$). 
Although significant negative correlations were plentiful, only `r abund_corr |> filter(q <= 0.05, estimate > 0) |> pull(taxon) |> n_distinct()` taxa had significant positive correlations 
with either butyrate, propionate or valerate (Figure 4B). 
Acidaminococcaceae_unclassified (OTU0007) was correlated with propionate at 28 (rho ($\rho$) = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age=="28 (8)" , SCFA == "Propionate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age=="28 (8)", SCFA == "Propionate") %>% pull(q) %>% round(.,3)`) and 35 days (rho ($\rho$) = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age=="35 (15)", SCFA == "Propionate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age == "35 (15)", SCFA == "Propionate") %>% pull(q) %>% round(.,3)`). Acidaminococcaceae_unclassified (OTU0007) 
was also positively correlated with valerate at 35 days (rho ($\rho$) = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age=="35 (15)", SCFA == "Valerate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Acidaminococcaceae_unclassified (Otu0007)", age=="35 (15)", SCFA == "Valerate") %>% pull(q) %>% round(.,3)`). Ruminococcacea_unclassified spp. (OTU0047)
was positively correlated with butyrate at 28 days (rho ($\rho$) = `r abund_corr %>% filter(taxon == "Ruminococcaceae_unclassified (Otu0047)", age == "28 (8)", SCFA == "Butyrate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Ruminococcaceae_unclassified (Otu0047)", age == "28 (8)", SCFA == "Butyrate") %>% pull(q) %>% round(.,3)`). 

Two OTUs were negatively correlated with propionate at 15 dpi: *Anaerotignum* (Otu0059) (rho ($\rho$) = `r abund_corr %>% filter(taxon == "Anaerotignum (Otu0059)", age == "35 (15)", SCFA == "Propionate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Anaerotignum (Otu0059)", age == "35 (15)", SCFA == "Propionate") %>% pull(q) %>% round(.,3)`) and  Lachnospiraceae_unclassified (Otu0071) (rho ($\rho$) = `r abund_corr %>% filter(taxon == "Lachnospiraceae_unclassified (Otu0071)", age == "35 (15)", SCFA == "Propionate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Lachnospiraceae_unclassified (Otu0071)", age == "35 (15)", SCFA == "Propionate") %>% pull(q) %>% round(.,3)`); both of which belong to the Lachnospiraceae family. *Anaerotignum* spp. (Otu0059) was also
negatively correlated with valerate at 2 dpi (rho ($\rho$) = `r abund_corr %>% filter(taxon == "Anaerotignum (Otu0059)", age == "22 (2)", SCFA == "Valerate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Anaerotignum (Otu0059)", age == "22 (2)", SCFA == "Valerate") %>% pull(q) %>% round(.,3)`)
as was Bacillales_unclassified (Otu0060) (rho ($\rho$) = `r abund_corr %>% filter(taxon == "Bacillales_unclassified (Otu0060)", age == "22 (2)", SCFA == "Valerate") %>% pull(estimate) %>% round(.,3)`, *p* = `r abund_corr %>% filter(taxon == "Bacillales_unclassified (Otu0060)", age == "22 (2)", SCFA == "Valerate") %>% pull(q) %>% round(.,3)`).
Acidaminococcaceae_unclassified (Otu0007) was positively correlated 
with propionate and valerate concentration and was increased in SE-challenged birds. No overlap was observed between propionate, butyrate and 
valerate correlated OTUs and taxa associated with the jGOS group.

Correlation analysis was also performed using OTU relative abundance (%) in place of 
bias-corrected OTU abundances (Figure S5). This approach identified 
`r OTU_corr |> filter(q <= 0.05) |> pull(OTU) |> n_distinct()` OTUs with 
significant positive or negative associations with either butyrate, propionate or valerate, compared 
with a total of `r abund_corr |> filter(q <= 0.05) |> pull(taxon) |> n_distinct()` 
OTUs identified using the previous approach. However, there was agreement 
between the approaches with `r n_distinct(corob$taxon)` OTUs identified as 
having similar significant correlations. Of the `r df |> filter(chal_fill == "Positive LFC") |> pull(taxon) |> n_distinct()` taxa found to be 
positively associated with SE-challenge using the ALDEx2 and ANCOM-based approaches, all were positively correlated with 
propionate or valerate at some stage. Similarly, of the `r df |> filter(diet_fill == "Positive LFC") |> pull(taxon) |> n_distinct()` taxa increased in the jGOS group, *Megamonas* spp. (Otu0006) 
and *Lactobacillus* spp. (Otu0061) were positively correlated with propionate and valerate.
However, `r df |> filter(diet_fill == "Negative LFC") |> pull(id)` that was
negatively associated with the jGOS diet showed no correlation with either butyrate, propionate or valerate.

```{r discussion, include=FALSE}
#propionate_28_mM <- scfa_long %>% 
#                        filter(SCFA=="Propionate",age==28) %>% 
#                        group_by(cohort) %>% 
#                        summarise(mean_mM = mean(Concentration)) 

propionate_35_mM <- scfa_long %>% 
                        filter(SCFA=="Propionate",age==35) %>% 
                        group_by(cohort) %>% 
                        summarise(mean_mM = mean(Concentration)) 

valerate_35_mM <- scfa_long %>% 
                        filter(SCFA=="Valerate",age==35) %>% 
                        group_by(cohort) %>% 
                        summarise(mean_mM = mean(Concentration)) 

error <- read_tsv(error_path) %>% 
                                 mutate(sum_err = Sequences*Errors) 

```
## Discussion
We have shown that supplementing the feed of juvenile chickens 
with GOS until the birds are 24-days-old shortens the time required for clearance 
of *S.* Enteritidis infection following an oral challenge at 20 days. Oral gavage 
with *Salmonella* Enteritidis did not result in overt intestinal disease,
as characterized by intestinal inflammation or reduced growth performance, but 
did generate increased concentrations of cecal propionate at 8 and 15 dpi in 
chickens raised on either control or jGOS diets relative to diet-matched 
mock-challenged controls. At 15 dpi the levels of propionate detected in *Salmonella*-challenged chickens 
were greater in chickens in the jGOS group (`r propionate_35_mM %>% filter(cohort=="jGOS x SE") %>% pull(mean_mM) %>% round(2)` mM) 
relative to *Salmonella*-challenged chickens on the control diet (`r propionate_35_mM %>% filter(cohort=="ctl x SE") %>% pull(mean_mM) %>% round(2)` mM); establishing a link between SE challenge
and cecal propionate that is augmented by, but not dependent on, juvenile exposure 
to supplementary GOS. In parallel valerate concentrations were observed to be 
greater in the *Salmonella*-challenged jGOS group (`r valerate_35_mM %>% filter(cohort=="jGOS x SE") %>% pull(mean_mM) %>% round(2)` mM) compared to *Salmonella*-challenged chickens on 
the control diet (`r valerate_35_mM %>% filter(cohort=="ctl x SE") %>% pull(mean_mM) %>% round(2)` mM).
As supplementary GOS was absent from the diet from 24 days old (4 dpi) this is likely a legacy effect of 
consuming GOS. Culture-based experiments have ascertained that 15 mM propionate 
at pH 6.5 will inhibit the growth of *S.* Typhimurium [@barnes1979factors]; 
whereas 25 mM propionate at pH 6 is required to inhibit the growth of 
*S.* Enteritidis [@vanImmerseel2003invasion]. Moreover, 3.13 mM propionate 
is reported to increase the lag time of *S.* Typhimurium and 6.25 mM propionate to decrease 
the growth rate [@jacobson2018propionate]. Valerate is also inimical to *Salmonella*
in culture with inhibitory concentrations for *S*. Typhimurium and *S*. Enteritidis 
reported between 25 and 37 mM [@kovanda2019organicacids; @lamas2019scfa]. Although 
the concentrations of the SCFAs we measured in the cecal contents are lower than 
required *in vitro*, it is possible that the inimical effect of propionate 
combined with valerate is heightened at the pH of the ceca or the niches occupied 
by *Salmonella*.  

Although propionate and butyrate are reported to have immunoregulatory activities
in mice [@arpaia2013metabolites], we did not find evidence of any host mediated 
responses characteristic for chickens in the jGOS group and therefore the direct 
action of the SCFAs propionate and valerate on *Salmonella* seems more likely. The 
mechanism by which propionate inhibits *Salmonella* growth has yet to be established, 
but the predominant hypothesis is that dissociated propionate diffuses across the 
cell membrane and acidifies the cytoplasm [@ricke2003perspectives; @jacobson2018propionate; @sorbara2019inhibiting].
The alternate, and possibly complementary hypothesis, is that at 10 mM 
propionate represses *S.* Typhimurium invasion through post-translational control 
of HilD [@hung2013HilD]. However, an extensive study by Shelton *et al.* 
[@shelton2022salmonella] showed that in the inflamed mouse gut *S.* Typhimurium 
can breakdown propionate into pyruvate via the 2-methylcitric acid cycle, which 
is encoded by the *prpBCDE* operon [@horswill1999methylcitric]. Studies using 
*S*. Typhimurium have indicated that activation of the *prpBCDE* operon is 
dependent on the availability of respiratory electron acceptors, such as oxygen 
in aerobic respiration [@palacios2003propionylcoa] or nitrate during anaerobic 
respiration [@shelton2022salmonella]. Whilst *S*. Enteritidis P125109 has an intact 
*prpBCDE* operon that can in principle metabolise propionate [@thomson2008comparative;@horswill1999methylcitric], 
the chicken cecal lumen maintains a flourishing anaerobic community due to the avid 
consumption of capillary oxygen by the enterocytes catabolizing butyrate generated 
by the microbiota, and in the absence of an inflammatory response to *S*. Enteritidis 
the electron acceptor nitrate will not be available to activate *prpBCDE* gene expression. 
Under these circumstances *S*. Enteritidis would not be able to prevent propionate 
toxicity.

In mice the prohibitory effect of luminal propionate on *Salmonella* is dependent 
on the immune response of the host since although propionate mediates colonization 
resistance to *S*. Typhimurium in conventionally colonized mice [@jacobson2018propionate], 
an extensive study by Shelton *et al.* [@shelton2022salmonella] showed that following 
streptomycin treatment, *Salmonella*-challenge resulted in inflammation in the
gut and under these conditions propionate can act as a carbon source for *S.* Typhimurium. 
Parallels of this can be recognized in the literature reporting *Salmonella* colonization of
chickens, whilst being aware of the established differences in virulence between 
*Salmonella* serovars [@foley2013serovar], virulence factor induced inflammation of 
1-day-old chicks has been shown to facilitate the outgrowth of *S.* Enteritidis 
by aerobic respiration [@litvak2019commensal] and a profile of *S.* Typhimurium 
gene expression in the cecum of chicks challenged at 12 h from hatch identified 
propionate as a carbon source [@harvey2011salmonella]. A comparative genomic 
study of gastrointestinal and extraintestinal pathovars of *Salmonella* revealed
the extraintestinal pathovars to have degraded the regulators for anaerobic 
processes including *prpR* - the propionate catabolism operon regulatory protein 
[@nuccio2014comparative]. The authors conclude the retention of these regulatory 
networks provides the gastrointestinal pathovars the opportunity to outcompete 
other microorganisms present in the lumen of the inflamed gut. However, the 
involvement of propionate in infection is likely complex since *Salmonella*
Gallinarum is an extraintestinal pathovar (the causative agent of systemic fowl 
typhoid in chickens) that retains the *prpR* gene but does not trigger a 
pro-inflammatory response in the gut, and is reported to be less likely to cause 
disease in chickens sustained on feed supplemented with propionate 
[@defreitasneto2013flagellated; @kaiser2000differential; @thomson2008comparative; @berchieri1996reduction]. 
Whereas micro-encapsulated dietary propionate did not afford protection to young 
chickens (5 d) exposed to *S.* Enteritidis that also possess the *prpR* gene
[@vanimmerseel2004microencapsulated]. Levels of valerate followed a similar 
trend to propionate and were also higher in *Salmonella*-challenged chickens, 
especially in the presence of GOS that led to the clearance of intestinal *S*. 
Enteritidis. Consistent with this observation chickens with low heterophil/lymphocyte 
ratios show resistance to *S*. Enteritidis and where the lower abundances of *S*. 
Enteritidis are reported to correlate with greater cecal propionate and valerate 
contents [@thiam2022heterophil]. There are, however, few reports of the origin of 
microbiota-generated valerate in the literature, nor of its effect on the host. 
Nevertheless, valerate has been shown to have anti-*Clostridium difficile* activity in mice 
[@mcdonald2018inhibiting]. No OTUs from the *Clostridium* cluster XI were found 
to discriminate between cohorts in this study, but *Anaerotignum* spp. (Otu0059) 
which falls within the broader grouping of the class Clostridia were negatively 
correlated with both cecal propionate and valerate concentrations.  We observed that 
*Salmonella* was more effectively cleared from the cecal pouches of chickens from 
the jGOS group, but that the organism persisted in the liver and spleen, and 
potentially other extra-intestinal tissues. Selective clearance of intestinal 
*Salmonella* with GOS, possibly in combination with other non-antibiotic feed 
additives [@sun2022kdf] may be exploited in future studies to determine the 
potential for the persistence and re-colonisation of the gut by extra-intestinal 
cells.

Amplicon profiling indicated that Acidaminococcaceae_unclassified (Otu0007) was 
associated with *Salmonella*-challenge and correlated with the levels of caecal
propionate and valerate. Querying the representative OTU sequence 
for Otu0007 using NCBI BLASTn [@altschul1990blast] revealed similarity to 
*Phascolarctobacterium faecium* (E value = 6e-110; % identity =	96.03%; accessed 
March 3 2024).
*P. faecium* has been shown to produce propionate through the catabolism of the
dicarboxylic acid succinate [@wu2017phascolarctobacterium]. Experiments using 
*in vitro* batch fermentation have shown that GOS is fermented to succinate by the
human microbiota [@hopkins2003nondigestible], suggesting that the increased levels 
of *P. faecium* are due to cross-feeding and not directly the catabolism of GOS. 
*Megamonas* spp. (Otu0006) was also associated with SE-challenge and a positive 
correlation was shown between its proportional abundance and the cecal concentrations 
of propionate and valerate. *Megamonas* spp. are also known to metabolize succinate and produce propionate 
[@polansky2016processes; @barnes1979factors; @sakon2008sutterella; @chevrot2008rupellensis] and
chickens dosed with a probiotic *Bacillus* cocktail (two *Bacillus subtilis* strains, 
plus *Bacillus amyloliquefaciens*) were reported to produce significant increases 
in fecal propionate and in the abundance of *Megamonas* [@khan2020salmonella]. 
Acidaminococcaceae_unclassified (Otu0007) and *Megamonas* spp. 
(Otu0006) are closely related taxa. The representative OTU sequences from this 
trial have 86.22% sequence identity, and on the basis of 16S rRNA gene sequence 
alignment it has been reasoned that the genus *Megamonas* be reclassified as a 
member of the family Acidaminococcaceae [@morotomi2007megamonas]. However, if the 
Acidaminococcaceae_unclassified (Otu0007) and *Megamonas* spp. (Otu0006) similarly 
benefit from the provision of substrate succinate, this leaves the identities of 
the primary GOS degraders still to be determined. The 16S rRNA sequencing error 
rate, as calculated against a known mock community (positive control) indicates 
that the errors introduced during sequencing and analysis are within appropriate 
margins. However, although *Salmonella* Enteritidis was successfully cultured 
from the cecal contents of all infected chickens and all rarefaction curves 
approached asymptote, *Salmonella* was not detected using 16S rRNA amplicon 
profiling. It appears that the levels of *Salmonella* colonizing the gut were 
below the limit of detection of this approach. 

A mature, healthy gut microbiota is a significant hurdle to *Salmonella* colonization 
in both chicken and mouse models [@rogers2020microbiome; @nurmi1973new; @stecher2010like]. 
It has been established that a protective effect against *Salmonella* colonization 
can be achieved by exposing young chicks (<7 days old) to the gut microbiota of 
mature birds [@nurmi1973new]. Deliberate endowment of colonization resistance to 
the chicks in intensive production systems has proved difficult as there is a risk 
of pathogen transfer from maternal birds and although competitive exclusion 
products of mixed cocktails of bacteria are commercially available for seeding 
the chick gut [@methner1997comparative; @nakamura2002evaluation], they have faced 
steep regulatory hurdles in some territories. However, a recent study reported 
increases in the abundance of Bacteroidetes in the intestines of juvenile chickens 
after the donation of mature cecal microbiota, and the highlighted the bacterial 
taxa that readily colonize juvenile ceca to include *Megamonas* and 
*Phascolarctobacterium* [@marcolla2023cecal] identified in the current study as 
responding to *S*. Enteritidis challenge with greater abundances evident on the 
jGOS diet. The juvenile birds inoculated with mature cecal contents also produced 
greater concentrations of cecal propionate and valerate [@marcolla2023cecal]. 
Taxa that can evolve SCFA in the gut to increase niche competition but require 
careful control [@pottenger2023timing]. The deliberate manipulation of colonization 
resistance factors by introduction of prebiotics is an attractive alternative to 
introduction of a novel microbiota.

We have previously reported that GOS improves the growth rate of Ross 308 broiler 
chickens [@richards2020gos]. In the results presented here dietary GOS did not 
increase the body weight at 28 or 35 days, which is likely due to the absence of 
GOS in the finisher diet (24-35 days) [@richards2020gos]. Previous studies that 
assessed the effect of supplementing feed with propionate on broiler performance 
noted a reduction in fat deposition and a reduction in average daily gain [@li2021propionate]. 
However, *in situ* generation of propionate by cecal microbiota in response to 
*Salmonella* challenge represents a more focused and potentially more dynamic 
exposure that may be less likely to give rise to wider physiological effects. 
Outside the scope of this study the association between *Salmonella* colonization 
in the absence of disease and growth performance merits further investigation.

Here we report evidence that indicates that the process by which dietary GOS hastens 
*Salmonella* clearance is connected to the biochemical activities of specific 
bacterial taxa. We hypothesize that supplementary dietary GOS is catabolized to 
succinate in the ceacal lumen, which cultivates increased populations of succinate utilizing 
bacteria. Succinate metabolism has been shown to have an important role in allowing 
*Salmonella* to colonize the mouse gut [@spiga2017succinate]. The deleterious 
effect on *S*. Enteritidis colonization is therefore likely two-fold: in the absence 
of inflammation the increased concentrations of propionate and valerate are 
inimical to *Salmonella* growth and increased competition for succinate presents 
an additional hurdle. However, the greatest relative abundance of Acidaminococcaceae 
and concentrations of propionate were detected after GOS supplementation of the 
feed had ceased implying that the early provision of GOS has a priming effect on 
the chicken cecal microbiome that enables the response to *Salmonella* colonization. 
Supplementing the diet of juvenile chickens with GOS increases the proportion of 
a narrow category of bacterial taxa and confers a specific benefit to the host, 
ie. hastening the eradication of intestinal *Salmonella* infection. Specific responses
of the intestinal microbiota to GOS fulfill the prebiotic criteria described by 
Gibson *et al.* [@gibson2017prebiotic] and is an attractive approach as a safe 
and effective intervention against a key zoonotic, foodborne pathogen.

# Materials and methods

## Chickens
Commercial male Ross 308 broiler chicks were obtained as hatchlings (PD Hook, UK) 
and housed under strict biosecurity in a controlled environment operating at 
containment level 2 (https://www.hse.gov.uk/biosafety/management-containment-labs.pdf) 
at the University of Nottingham. Ambient temperature and relative humidity were 
controlled as dictated in the Code of Practice for the Housing and Care of 
Animals Bred, Supplied or Used for Scientific Purposes. Feeds were formulated on 
a least cost basis to meet the requirements set out in the Ross 308 Broiler 
Nutrition Specifications 2014 (Aviagen, UK) and prepared by Target Feeds Ltd 
(Shropshire, UK). Throughout the trial feed and water were provided *ad libitum*. 
In order to ensure that the birds were *Salmonella* free upon arrival, the 
absorbent papers on which the birds had been transported were shredded and then 
added to Buffered Peptone Water (Oxoid, UK) for *Salmonella* enrichment as 
described below.

## Feeding regime
The feeding regime was as follows: the control diet group was sustained on a 
wheat-soybean meal-based feed provided as a starter mash at 0-10 days, grower 
pellets at 11-24 days and finisher pellets at 25-35 days. The starter diet contained 
wheat (61.5% w/w), soya meal (31.1% w/w), soyabean oil (3.39% w/w), limestone 
(0.8% w/w), calcium phosphate (1.25% w/w), sodium bicarbonate (0.335% w/w), the 
enzymes phytase and xylanase (dosed according to the manufacturer's instructions 
(DSM Nutritional Products Ltd., CH) and a vitamin mix containing salt, lysine HCl,
DL-methionine and threonine. The grower and finisher diets increased the wheat 
content at the expense of soya meal by 1 and 6% w/w respectively. Figure 1 
summarizes the timings of changes in feed composition, *Salmonella* challenge 
and sample collection. GOS was provided as Nutrabiotic® GOS 64%, 
batch no. AQ8035 (Saputo Dairy, UK). Galactooligosaccharide preparations are 
manufactured by the $\beta$-galactosidase catalysed transglycosylation of lactose 
producing, in the case of batch AQ8035: glucose and galactose (22.4 % m/m DM), 
lactose (10.1 % m/m DM) and other disaccharides that are functionally included 
as galactooligosaccharides, and galactooligosaccharides (DP2 to DP8+ and typically 
of the form (gal)n-glc). Nutrabiotic® GOS 64% is produced as a feed grade syrup; 
batch AQ8035 has a dry matter concentration of 73.4% m/m and a total 
galactooligosaccharides concentration of 66.5% m/m DM. Starter feed was 
supplemented with 2.32% w/w GOS and isocaloric adjustments made in the wheat and 
soybean oil contents. The grower feed contained 1.16% GOS with the wheat and 
soybean oil contents adjusted accordingly. Nutrabiotic® GOS was not added to the 
finisher feed. The final feeds were isocaloric with the same content of crude 
protein and Degussa poultry digestible amino acid. The feed formulations are 
listed in supplementary Table S3.

## Challenge trial
*Salmonella enterica* serovar Enteritidis P125109 [@thomson2008comparative]
was selected as a challenge strain as the nalidixic acid resistance it exhibits 
facilitates enumeration through direct cultivation. The challenge trial was 
performed to determine the effects of *Salmonella* exposure on broiler chickens 
sustained on GOS-supplemented feed, where the experimental implementation and 
husbandry were conducted according to the ARRIVE guidelines [@kilkenny2010ARRIVE]. 
Upon arrival chicks were weighed and randomly placed in pens of either 5 or 6 
birds in one of two environmentally controlled containment rooms. Each room 
contained 14 independent pens with 7 pens per experimental group designated to 
receive either the control or GOS-supplemented feed. Samples were subsequently 
collected from a single chicken from each pen at each experimental timepoint.
The coprophagous behavior of co-housed chickens means the birds are exposed to 
bacteria from each other's intestinal content and therefore in this design the 
pen is the independent experimental unit (*n* = 7). At 
20-days-old chickens in one room were orally challenged with 4.6 x 10^8^ CFU  
*S.* Enteritidis suspended in 1 ml of Maximum Recovery Diluent (MRD; Oxoid, UK)
or in the second room given an equivalent mock dose of sterile MRD. Following 
challenge samples were collected at 2, 4, 8 and 15 days post-infection. 

Live weights were recorded for the chickens selected at random from each pen 
for each time point, and the birds euthanized by exposure to rising CO~2~ or 
parenteral barbiturate overdose followed by cervical dislocation in accord with 
Schedule 1 of the UK Animals (Scientific Procedures) Act. Sample tissues, 
cardiac blood and intestinal contents were collected immediately post-mortem. 
Ileal tissues were sectioned from approximately 3 cm distal to Meckel's 
diverticulum and cecal tissues from the distal tips of the ceca. Intestinal 
tissues were either flash frozen in liquid nitrogen for subsequent RNA isolation 
or preserved in 10% (w/v) neutral buffered formalin (Fisher Scientific, UK) for 
histological assessment. Approximately 1 g of cecal contents was aseptically 
collected from the distal end of each cecal pouch and flash frozen in liquid 
nitrogen before being stored at -80°C prior to genomic DNA isolation. A further 
2-3 g sample of cecal contents was similarly collected from both ceca and combined 
in pre-weighed Universal vials before being stored on ice prior to *Salmonella* 
enumeration and SCFA analysis. Prior to incising the gut, the spleen and liver 
were aseptically excised with tissue samples preserved in 10% (w/v) neutral 
buffered formalin and the remainder placed in sterile polythene bags and stored
on ice prior to *Salmonella* enumeration.

## *Salmonella* enumeration
Cecal contents were suspended at 0.1 g/ml and serially diluted 1:10 in MRD from
which 100 $\mu$l aliquots over a range of dilutions were spread onto XLD agar 
(Oxoid, UK) modified with 1 $\mu$g/ml Novobiocin (Sigma-Aldrich, UK) and 12.5 
$\mu$g/ml Nalidixic acid (Sigma-Aldrich). Cecal contents collected from each 
euthanized chicken were plated in triplicate and incubated at 37°C for 24 h prior 
to enumeration. Mean values of log~10~-transformed colony-forming units were used 
for statistical analysis. *Salmonella* was detected through enrichment by 
transferring ~0.1 g cecal contents or chick bedding papers to 10 ml BPW that was then 
incubated at 37°C for 16-20 h before 0.1 ml volumes of the BPW suspension were 
dispensed onto MSRV agar (Oxoid) in triplicate. MSRV plates were incubated without 
inversion at 42°C for 24 h before any growth was sub-cultured to XLD plates (with 
no nalidixic acid or novobiocin added) and incubated at 37°C. Presumptive 
*Salmonella* colonies were confirmed with Poly O antiserum (Pro-Lab Diagnostics, 
UK). Liver and spleen tissues (1 - 5 g) were suspended 1:10 w/v in BPW and 
disrupted in a Seward Stomacher 80 (Seward Biomaster, UK) for 1 min on medium 
speed. Viable *Salmonella* present were recovered by direct plating 
of 0.1 ml of the tissue suspension onto modified XLD or through enrichment by 
incubating the remaining tissue suspension followed by isolation of any 
*Salmonella* present on MSRV. Incubation conditions and confirmation methods are 
as described above.

## 16S rRNA gene sequences and analysis
Total DNAs were isolated from cecal contents using a QIAcube HT apparatus and 
QIAamp PowerFecal Pro DNA kit (Qiagen, UK). Bacterial 16S rRNA gene sequences
were PCR amplified using universal primers: 515f (5' GTGCCAGCMGCCGCGGTAA 3') 
and 806r (5' GGACTACHVGGGTWTCTAAT 3') designed to flank the hypervariable V4 
region of the 16S rRNA genes [@caporaso2011global]. Amplicons were sequenced 
on the Illumina MiSeq platform (Illumina, USA) using 2 × 250 bp cycles according 
to the manufacturer’s instructions and the protocol described by Kozich *et al.* 
[@kozich2013sop]. Using the same protocol, no-template negative control amplicon 
libraries were prepared for both kit reagent and PCR batches and a positive control
library was prepared from a defined mock community (ZymoBIOMICS Microbial Community 
DNA Standard, Zymo Research, USA). Mothur code to reproduce the analyses has been 
made available (see Data and code availability statement). All sequence data of 
the 16S rRNA genes were quality filtered and then clustered into operational 
taxonomic units (OTUs) in mothur v1.48 [@schloss2009introducing] 
using the MiSeq as described by the package developers [@kozich2013sop].

## Host gene expression analysis
Chicken intestinal total RNA was isolated from ileal and cecal tonsil tissues. 
Tissue biopsies were weighed out to 100 mg followed by homogenization with 
RLT lysis buffer supplied with the RNeasy kit (Qiagen, Germany) and 2.8 mm 
ceramic beads (MO BIO Laboratories Inc., Canada) in a TissueLyser II (Qiagen) 
bead mill. Total RNA was then extracted from lysates using the RNeasy kit and 
QIAcube HT (Qiagen). The quality of the extracted RNAs were analyzed using a 
TapeStation 2200 (Agilent, USA) by measuring the RNA integrity number. Reverse
transcription of extracted RNA was carried out with the RT2 First Strand Kit 
(Qiagen) to synthesise the first-strand cDNA and remove any residual genomic DNA 
according to manufacturer’s instruction. The expression of host cytokine and 
chemokine genes were determined by qPCR with RT2 SYBR Green qPCR Mastermix 
(Qiagen) in a Roche Diagnostics Light Cycler 480 instrument (Hoffmann La Roche AG, 
Switzerland). Primer sets were purchased as customized RT2 Profiler PCR 
Arrays (Qiagen) and verified to detect the transcripts for IFN-$\gamma$, IL-1$\beta$, IL-4, 
IL-6, IL-8L1, IL-8L2, IL-9, IL-10, IL-13, IL-15, IL-17A, IL-22, NOS2, STAT3, STAT4, STAT6, 
STAT5$\beta$, GATA3 and TGIF1 in chicken. The means of triplicate Ct values were 
used for analysis, where target genes Ct values were independently normalized to 
those of the housekeeping genes glyceraldehyde 3-phosphate dehydrogenase (GAPDH) 
and 60S ribosomal protein L4 (RPL4). The normalised Ct values for the two 
housekeeping genes were in close agreement with an R2 of 0.94 across all 
determinations for cecal and ileal tissues.   

## Histology
Tissue samples were fixed in a 10% formalin solution and dehydrated through a 
series of alcohol solutions, cleared in xylene, and embedded in paraffin wax 
(Microtechnical Services Ltd, Exeter, UK). Sections (3 to 5 µm thick) were prepared 
and stained with modified hematoxylin and eosin (H&E). Independent intestinal 
sections were stained with periodic acid-Schiff (PAS). The stained slides were 
scanned using a NanoZoomer Digital Pathology System (Hamamatsu, Welwyn Garden City, 
UK) at 40X resolution. The liver sections were examined for the infiltration of 
inflammatory cells around a central vein, whether porta hepatis or sinusoid. 
Villus heights and crypt depths of intestinal tissues were recorded from operator 
blinded measurements of histological stained slides scanned for each tissue sample. 
Villus heights were measured from the tip of the villus to the crypt opening and 
the associated crypt depth was measured from the base of the crypt to the level 
of the crypt opening. The ratios of villus height to relative crypt depth (v/c 
ratio) were calculated from these measurements. Dimensions for 10 well-oriented 
villi per tissue sample of 7 birds per experimental group at each sampling time 
were analyzed.

## Immunoglobin quantification
Anti-*Salmonella* IgY and IgA antibodies in chicken serum were detected by 
enzyme-linked immunosorbent assay (ELISA) using the method of Withanage *et al.*
[@withanage2005cytokine] with the following modifications. *Salmonella* 
Enteritidis P125109 cells were subjected to 3 freeze–thaw cycles between 
room-temperature and -80°C, followed by sonication on ice using 9 x 20 s bursts
to break the cells. Unbroken cells were removed by centrifugation at 4000 x g 
for 20 min at 4 °C followed by filtration through a 0.2 $\mu$m filter. The protein
concentration of the resultant soluble fraction was measured using Pierce BCA kit
(Thermofisher, UK) and aliquots stored at -20 °C. For ELISA chicken serum 
samples were diluted 1:250 for IgY and 1:100 for IgA with PBS containing 
Tween 20 (0.05%) (PBS-T) and 3% skimmed milk powder (SMP). Horse radish 
peroxidase (HRP) conjugated rabbit anti-chicken IgY (Sigma Aldrich) and goat
anti-chicken (Abcam, UK) were diluted 1:5000 in PBS-T 3% SMP. The enzyme 
substrate used was 3,3´,5,5´-tetramentylbenzidine (TMB; Thermofisher) with the 
reaction stopped with 100 $\mu$l 2 M sulphuric acid after 30 min and absorbance read 
at 450 nm.

## SCFA quantification
Duplicate samples were analysed for the short chain fatty acids (SCFA): 
acetic, ($\ge$ 99%; Thermofisher), propionic, (99%; Thermofisher), butyric, ($\ge$ 99%;
Thermofisher), isobutyric, ($\ge$ 99%; Thermofisher), 2-methylbutyric, (>95%; 
TCI UK Ltd., UK), valeric, (99%; Thermofisher), isovaleric, (99%; Thermofisher), 
hexanoic, (98%; TCI UK Ltd.) and lactic acid, (90.2%; Thermofisher) by GC MS. 
Samples were prepared from weighed thawed cecal content material by sonication 
with the addition of 2-methylvaleric acid (>98%, Thermofisher) as internal 
standard. The SCFAs were converted to the ethyl ester and extracted into pentane 
(AR Grade; Thermofisher) along with the additions of the corresponding internal
standards. The internal standards were separately synthesised as the deuterated 
ethyl ester for each SCFA, except for ethyl lactate D5-benzoate the internal 
standard for lactic acid analysis. Sample SCFA concentrations were measured by 
GC-MS (Agilent 6890 + 5977 GC-MS; Agilent DB 5MS UI, 15 m x 0.25 mm x 1.00 $\mu$m 
column; He carrier gas at 1.20 mL/min.; using a defined oven temperature profile 
over a run time of 33.2 min.; operated in selected ion monitoring (SIM) mode 
with 10 different SIM groups) with reference to a standard curve for each SCFA, 
with each calibrant being prepared using a procedure similar to that for the 
samples. 

## Statistical analysis
Significant differences in the cecal burdens of *S. Enteritidis* between the 
challenged groups were tested using the Wilcoxon rank sum test with p-values $\leq$ 
0.05 indicating significance. Significant differences in body weights at each 
timepoint was determined using the Kruskal-Wallis test. For those timepoints with 
a *p*-value below a significance threshold of 0.05, a subsequent pairwise Wilcox 
test with Benjamini–Hochberg adjustment for multiple comparisons was performed 
to test for significance between cohorts. For alpha diversity as represented by 
the inverse Simpson index or Shannon entropy, statistical significance was determined 
using the Kruskal-Wallis test for each timepoint. For those timepoints with a 
*p*-value below a significance threshold of 0.05, a Dunn's test of multiple 
comparisons with Benjamini-Hochberg adjustment was performed to determine 
significance between cohorts. Differences in expression for each pair of expression 
comparisons per gene per timepoint were determined using Student's t-test with 
Benjamini-Hochberg adjustment. The ELISA results for the anti-*Salmonella* serum 
antibodies were analyzed using the Student’s t-test by comparing the *Salmonella* 
colonized birds with the non-colonized controls on the same diet for each timepoint.
For concentrations of each SCFA measured at each timepoint, statistical 
significance was determined using the Kruskal-Wallis test, for those SCFA with a 
*p*-value below a significance threshold of 0.05, a subsequent pairwise Wilcox 
test with FDR correction was performed to determine significance between cohorts. 
Correlations between paired observations of SCFA concentration and bacterial 
abundance were estimated using Spearman's $\rho$ statistic in R. An analysis was 
made using concentrations of each SCFA measured against either bias-corrected or 
proportional taxonomic abundance. The *p*-values detected for each SCFA and each 
timepoint were adjusted using Benjamini Hochberg correction. Statistical 
significance in villus length between cohorts was determined using the Tukey 
Honest Significant Difference method with adjustment for the multiple comparisons 
per timepoint. Differential abundance analysis of bacterial taxa was performed 
using the ALDEx2 (version 1.30.0) and ANCOM-BC (version 2.0.1) packages in R. 
ALDEx2 was performed using the aldex.clr modular approach. R code to reproduce 
both ALDEx2 and ANCOM-BC analysis has been made available on Github (https://github.com/PJRichards/richards_salmonella_gos).

All statistical analyses were performed in the R language (version 4.3.2) and all 
analyses were performed within a conda environment (https://docs.conda.io/en/latest/).
The .yml recipe for the conda environment has been made available on Github (https://github.com/PJRichards/richards_salmonella_gos)
alongside the R code to reproduce all analyses and figures.
The design for Figure 3C, D and E was adapted from Wallen *et al* [@wallen2022metagenomics].

## Data Availability
Amplicon sequence data have been deposited at NCBI and are publicly available as 
of the date of publication under the accession number PRJNA972431. All original 
code has been deposited on Github (https://github.com/PJRichards/richards_salmonella_gos). 
Any additional information required to reanalyze the data reported in this paper 
is available on Github (https://github.com/PJRichards/richards_salmonella_gos) 
or is available on request.

## Abbreviations
BPW: Buffered peptone water; da: Days of age; dpi: Days post infection; 
FCR: Feed conversion ratio; GOS: galactooligosaccharides; 
MRSV: Modified semi-solid Rappaport-Vassiliadis medium; 
OTU: Operational taxonomic unit; PCoA: Principal coordinate analysis; 
SCFA: Short chain fatty acids; SE: *Salmonella* Enteritidis; 
XLD: Xylose lysine deoxycholate culture medium.

## Declarations

### Ethical approval
The use of birds in experiments were subjected to the approval process under national 
guidelines by the United Kingdom Home Office. This project was performed under 
license of the United Kingdom Government Home Office Project Licensing ASPA 86. 
The University Ethics Committee internally reviewed all project licenses prior to 
submission to the Home Office. This includes the handling of animals, scrutiny of 
welfare and ethics.

## Acknowledgments
The authors acknowledge research funding to IFC from Saputo Dairy UK (formerly 
Dairy Crest Ltd). NMF acted as a consultant to Saputo Dairy UK. The authors thank 
Matt Stirling, at Analytical Innovations Ltd, who worked on SCFA method development 
and analysis. Authors contributions: PJR, PLC, NMF and IFC designed the trial. 
PJR, AA, LL, GMF, EK, AJC, PLC and IFC contributed to the sample collection and 
*Salmonella* enumeration. LL and AJC contributed to gene expression experiments. 
AA and EK contributed to the microbiota experiments. GMF and AJC contributed to 
the histology experiments. NMF contributed to the SCFA experiments. PJR and IFC 
analyzed the data. PJR and IFC wrote the manuscript.

## Figure and Table legends
```{r n calc, include=FALSE}
pcoa_n <- pcoa %>% 
                group_by(Trial,Challenge,Age,Feed) %>% 
                count()

alpha_n <- alpha %>% 
                    group_by(Age,Trial,Diet,Challenge) %>% 
                    count()

GE_n <- alDE %>% group_by(Feed,Challenge,Age,gene) %>% count()

sal_count_n <- SE_counts_raw %>% group_by(Age,Challenge, Diet) %>% count()

villus_n <- villus_len %>% 
            group_by(Age, Diet, Challenge) %>% 
            summarise(n = n()) 

```

**Figure 1. Trial design.** A total of `r alpha %>% nrow()` chickens were placed 
in 28 pens of 4 evenly divided between 4 treatment groups (ctl x mock, control 
diet with mock *S*. Enteritidis colonization; ctl x SE, control diet with *S*. 
Enteritidis colonization; jGOS x mock, GOS supplemented diet with mock *S*. 
Enteritidis colonization; jGOS x SE, GOS supplemented diet with *S*. Enteritidis 
colonization). Samples were collected by culling a single bird collected from each 
pen at each experimental timepoint.

**Figure 2. GOS-supplemented feed in early life hastens the clearance of *S.* Enteritidis from chickens.** 
Birds were challenged with *S.* Enteritidis at 20 days of age. The fill color indicates 
the diet: unfilled, control feed without GOS; yellow, feed supplemented with GOS. 
The shaded strip areas above each panel indicate the ages of the bird in days 
followed by days post infection in parentheses. 
**(A)** Cecal burden of *S.* Enteritidis. Each data marker represents an 
individual cecal community/bird collected from each independent pen (*n* = `r min(sal_count_n$n)`). 
The number above bar indicates *p*-value, determined using Wilcoxon rank sum test. 
Only *p*-values < 0.05 are shown. The upper dashed line indicates limit of direct 
enumeration, the lower dashed line indicates limit of detection by enrichment. 
**(B)** Chicken body weight for each of 7 pens. Significance was determined 
using the Kruskal-Wallis test for body weights measured at each timepoint. For 
those ages with a *p*-value below a significance threshold of 0.05, a subsequent 
pairwise Wilcox test with Benjamini–Hochberg adjustment for multiple comparisons
was performed to determine significance between cohorts. The number above the bar 
indicates the adjusted *p*-value. Only adjusted *p*-values < 0.05 are shown.
The dashed line indicates breed performance objective for male Ross 308 chickens 
(2014). 

**Figure 3. A distinct microbiota distinguishes *S.* Enteritidis-challenged chickens raised on a GOS-supplemented diet.**
**(A)** Alpha diversity as represented by inverse Simpson index. Each data marker 
represents individual cecal community/bird collected from each independent pen (*n* 
= `r min(alpha_n$n)`). The shaded strip areas indicate the bird age in days 
followed by days post infection in parentheses. Statistical significance was 
determined using the Kruskal-Wallis test for inverse Simpson indices measured at 
each timepoint, for those timepoints with a *p*-value below a significance 
threshold of 0.05, a Dunn's test of multiple comparisons with Benjamini-Hochberg 
adjustment was performed to determine significance between cohorts. The number 
above the bar indicates *p*-value. Only *p*-values < 0.05 are shown. The fill 
color indicates the diet at the time of sampling: unfilled, control feed without 
GOS; yellow, GOS supplemented feed.
**(B)** PCoA of Bray-Curtis dissimilarities (r^2^ = 0.55%). Each data marker 
represents an individual cecal community/bird collected from each independent 
pen (*n* = `r min(pcoa_n$n)`). Shaded eclipses indicate 95% confidence levels
calculated using ggpubr [@ggpubr] as per the FactoMineR package [@FactoMineR].
For each cohort (ctl is the control diet and jGOS the juvenile GOS supplemented 
diet) with those shaded grey mock colonized and those shaded red colonized with 
*S*. Enteritidis. The shaded strip areas above each panel indicate the ages of
the bird in days followed by days post infection in parentheses. The sequencing 
error rate was determined as `r 100*(sum(error$sum_err)/(sum(error$Sequences)*250)) |> round(4)`% 
using a control cocktail of 8 bacterial 
species (Sum of mismatches to reference / Sum of bases in query). 
**(C)** Discriminative taxa in SE challenged chickens identified using 
multivariate analysis in both ALDEX2 (version 1.30.0) and ANCOM-BC (version 2.0.1) packages. 
Superfluous '0's in OTU ID have been removed for clarity. OTUs were
identified as follows: `r df %>% filter(chal_fill != "p.adj > 0.05") %>% mutate(legend = paste(taxon, "=", id)) %>% pull() %>% str_c(collapse = "; ")`. 
**(D)** Discriminative taxa in jGOS-raised chickens identified using multivariate analysis 
in both ALDEX2 and ANCOM-BC packages. Superfluous '0's in OTU ID have been removed 
for clarity. OTUs are identified as follows: `r df %>% filter(diet_fill != "p.adj > 0.05") %>% mutate(legend = paste(taxon, "=", id)) %>% pull() %>% str_c(collapse = "; ")`
**(E)** Effect of S. Enteritidis challenge on expression of IL-17A, IL-22, 
IL-8L1, IL8-L2, IFN-$\gamma$ and IL-1$\beta$ in the ceca. Gene expression calculated 
as 2^-($\Delta\Delta$Ct)^ and presented as fold-change in *Salmonella*-challenged 
chickens relative to the mean expression level of mock-challenged chickens for 
each feed group (Control or jGOS). Note that *n* = `r GE_n |> filter(gene %in% c("'IL-17A'","IFN-gamma","'IL-22'","'IL-8L2'", "'IL-8L2'", "'IL-8L1'")) %>% pull(n) %>% max()`
for all groups, excluding the control diet, mock-challenged cohort value for IL-17A 
at 22 days old (*n* = `r GE_n %>% filter(gene %in% c("'IL-17A'","IFN-gamma","'IL-22'","'IL-8L2'", "'IL-8L2'", "'IL-8L1'")) %>% pull(n) %>% min()`). The number above the bar 
indicates the adjusted *p*-value as determined using Students t-test for each 
pair of expression comparisons per gene per time point with FDR correction 
performed across each time point with the Benjamini-Hochberg method. Only 
*p*-values < 0.05 are shown. Shaded strips above each panel indicate
the corresponding gene name.
**(F)** Ileal villus length per individual bird collected from each independent pen
(*n* = `r min(villus_n$n)`). Statistical significance was determined using 
Tukey Honest Significant Difference method with adjustment for the multiple 
comparisons per timepoint. No *p*-values are shown on the figure as all were in
excess of a significance threshold of 0.05.

**Figure 4. Relationship of microbiota to SCFA profile.** 
**(A)** Concentration of butyrate, propionate and valerate in the cecal lumen. The
plot shows the median concentration of `r scfa_long %>% filter(!str_detect(Group, pattern = "l_ctl_unc_35_\\d|l_gos_sal_22_\\d|l_gos_sal_35_\\d") ) %>% group_by(cohort, age, SCFA) %>% count() %>% pull(n) %>% unique()` 
individual cecal community/bird collected 
from each independent pen, apart from ctl x Mock at 35 days (*n* = `r scfa_long %>% group_by(cohort, age, SCFA) %>% count() %>% filter(cohort=="ctl x Mock", age=="35") %>% pull(n) %>% unique()`) 
and jGOS x SE at 22 and 35 days (*n* = `r scfa_long %>% group_by(cohort, age, SCFA) %>% count() %>% filter(cohort=="jGOS x SE", age=="22" | age=="35") %>% pull(n) %>% unique()`) 
due to difficulties in extracting the sample. Plot 
error bars indicate the upper (97.5th) and lower (2.5th) percentiles. Statistical 
significance was determined using the Kruskal-Wallis test for each SCFA measured at
each timepoint, for those SCFA with a *p*-value below a significance threshold of
0.05, a subsequent pairwise Wilcox test with FDR correction was performed to determine 
significance between groups. Asterisk symbol above the bar indicates pairwise Wilcox 
test *p*-value. For clarity significance is indicated by asterisk above bar: *p* 
< 0.001, \*\*\*; *p* < 0.01, \*\*; *p* < 0.05, \*. 
**(B)** Correlation plot showing significant relationships between bias-corrected 
OTU abundances and absolute concentration (ppm) of the SCFA metabolome. 
Both size and color of the data marker indicate Spearman correlation coefficient with
FDR correction for each SCFA at each time point. Filled data markers indicate significance 
(*p* $\leq$ 0.05). Results are reported at OTU taxonomic 
level. Due to failure to measure SCFA in some samples, outlined above, the 
correlations were calculated on the basis of `r abund_scfa |> filter(age=="22 (2)") %>% pull(Group) |> n_distinct("Group")` measurements at 22 days, `r abund_scfa |> filter(age=="24 (4)") %>% pull(Group) |> n_distinct("Group")`
measurements at 24, `r abund_scfa |> filter(age=="28 (8)") %>% pull(Group) |> n_distinct("Group")`
measurements at 28 days and `r abund_scfa |> filter(age=="35 (15)") %>% pull(Group) |> n_distinct("Group")` measurements at 35 days.

**Figure S1. Sequencing effort for all communities.** Rarefaction curves generated
using re-sampling without replacement. 

**Figure S2. Supplementing feed with GOS in early life affects changes in the cecal microbiota caused by *Salmonella* challenge**
Alpha diversity is described using Shannon Entropy. Each data marker represents 
individual cecal community/bird collected from each independent pen (*n* 
= `r min(alpha_n$n)`). The shaded strip areas indicate the bird age in days 
followed by days post infection in parentheses. Statistical significance was 
determined using the Kruskal-Wallis test for inverse Simpson indices measured at 
each timepoint, for those timepoints with a *p*-value below a significance 
threshold of 0.05, a Dunn's test of multiple comparisons with Benjamini-Hochberg adjustment
was performed to determine significance between cohorts. The number above the bar 
indicates *p*-value. Only *p*-values < 0.05 are shown. The fill color indicates the diet 
at the time of sampling: unfilled, control feed without GOS; yellow, GOS supplemented feed.

**Figure S3. Effect of *S.* Enteritidis challenge on expression of genes connected to innate immune response.** 
Effect of *S.* Enteritidis challenge on cecal expression of GATA3, IL-10, IL-15,
IL-4, NOS2, STAT3, STAT4, STAT5B, STAT6 and TGIF1. Gene expression calculated as 
2^-$\Delta\Delta$Ct^ and presented as fold-change in *Salmonella*-challenged 
chickens relative to the mean expression level of mock-challenged chickens for each 
feed group (Control or jGOS). Note that *n* = `r GE_n %>% filter(gene %in% c("IL4", "STAT6", "STAT5B", "STAT3", "IL10", "NOS2", "STAT4", "GATA3", "IL15", "TGIF1"))  %>% pull(n) %>% unique()`
for all groups presented here. The number above the bar indicates the adjusted 
*p*-value as determined using Student's t-test for each pair of expression 
comparisons per gene per time point with FDR correction performed across each 
time point with the Benjamini-Hochberg method. Only *p*-values < 0.05 are 
shown. Shaded strips above each panel indicate the corresponding gene name. 

**Figure S4. ELISA detection of *S*. Enteritidis specific serum antibodies from chickens infected with *S*. Enteritidis or mock-infected and raised on either a control or jGOS-supplemented diet.** 
**(A)** IgY isotype anti-*Salmonella* antibody detection (serum diluted 1:250). 
**(B)** IgA isotype anti-*Salmonella* antibody detection (serum diluted 1:100). 
These data are recorded as the means of the optical density readings from the ELISA 
at 450 nm for seven birds per group at each time point (days of age). The error 
bars indicate the standard errors of the means. Bird age in days is shown on the x-axis
with days post infection in parentheses. The number above the bar indicates the 
*p*-values determined using the Student’s t-test by comparing *Salmonella* 
colonized group with the non-colonized controls on the same diet for each timepoint.

**Figure S5. Correlation plot showing significant relationships between proportional OTU abundances and absolute concentration (ppm) of the SCFA metabolome.** 
Correlation plot showing significant relationships between proportional
OTU abundances and absolute concentration (ppm) of the SCFA metabolome. 
Both size and color of data marker indicate Spearman correlation coefficient with
FDR correction for each SCFA at each timepoint. Filled data markers indicate significance 
(*p* $\leq$ 0.05). Results are reported at OTU taxonomic levels. 
Due to failure to measure SCFA in the samples outlined in Figure 4, correlations 
were calculated on the basis of `r abund_scfa |> filter(age=="22 (2)") %>% pull(Group) |> n_distinct("Group")` measurements at 22 days, `r abund_scfa |> filter(age=="24 (4)") %>% pull(Group) |> n_distinct("Group")`
measurements at 24, `r abund_scfa |> filter(age=="28 (8)") %>% pull(Group) |> n_distinct("Group")`
measurements at 28 days and `r abund_scfa |> filter(age=="35 (15)") %>% pull(Group) |> n_distinct("Group")` measurements at 35 days.

**Table S1. Prevalence of *Salmonella* Enteritidis in liver and spleen tissues following oral challenge at 20-days old.**
The table shows the number of liver and spleen tissues collected from 
challenged chickens from which *Salmonella* could be isolated. The proportion of
chickens is reported in parentheses.  

**Table S2. Abundance of discriminative OTUs observed in the no-template control kitome.**
The table shows the number of reads that clustered with each discriminative OTU 
for each of the no-template control communities. Kitomes 1, 2 and 3 were prepared 
using different batches of DNA isolation kit reagents used for sample preparation. 
The figures in parenthesis indicate the rank of the read abundance for that OTU 
compared to all `r nrow(OTU_raw)` communities surveyed, with `r nrow(OTU_raw)` 
being the highest rank and 1 the lowest. Tied values are jointly assigned the 
highest rank.

**Table S3. GOS and control feed formulations.** Feed formulations for starter 
(0-10 days of age), grower (11-24 days of age) and finisher (25-35 days of age) diets.

## References
  